modules/appeals_api/app/swagger/appeals_api/v2/swagger.json

```diff
@@ -6745,6 +6746,26 @@
                                   "email": "bobsemail@bobbytablesemail.com",
                                   "timezone": "America/Chicago"
                                 },
+                                "claimant": {
```

Comment:
Since SC NVCs are still in progress, we don't want to put anything into this production doc regarding SC NVCs. Let's remove this addition.

Reply:
😡